src/installer/installer.go

```diff
@@ -436,9 +437,46 @@ func (i *installer) waitForControlPlane(ctx context.Context) error {
 		return err
 	}
 
+	i.waitForETCD(ctx, kc)
 	return nil
 }
 
+func (i *installer) waitForETCD(ctx context.Context, kc k8s_client.K8SClient) {
+	// Check Openshift version. ETCD configmap introduced in OCP 4.12+
+	v1, err := version.NewVersion(i.OpenshiftVersion)
+	if err != nil {
+		i.log.WithError(err).Warnf("failed to create new version with %s", i.OpenshiftVersion)
+		return
+	}
+	constraints, err := version.NewConstraint(">= 4.12")
+	if err != nil {
+		i.log.WithError(err).Warnf("failed to create new version constraint")
+		return
+	}
+	if !constraints.Check(v1) {
+		i.log.Infof("Detected OCP version is %s, skipping wait for ETCD bootstrap, constraint: %s", i.OpenshiftVersion, constraints.String())
+		return
+	}
+	i.log.Infof("Detected OCP version is %s. Waiting for ETCD bootstrap to be complete", i.OpenshiftVersion)
+	//TODO: Should it use a pre-existing status or should a new one be created?
+	i.UpdateHostInstallProgress(models.HostStageWaitingForBootkube, "waiting for ETCD bootstrap to be complete")
+
+	for {
+		select {
+		case <-ctx.Done():
+			i.log.Infof("Wait for bootstrap ETCD has been cancelled")
+			return
+		case <-time.After(generalWaitInterval):
+			// check if bootstrap etcd configmap has been created every minute
```

Comment:
The comment seems misleading, the `generalWaitInterval` interval is 5 seconds not a minute

Reply:
Thanks for catching that! Sorry meant to make it actually one minute